lib/DDG/Rewrite.pm

```diff
@@ -186,13 +191,26 @@ sub _build_nginx_conf {
 	# When we get errors from the endpoint, instead of replying a blank page, it should reply the function instead with no parameters,
 	# e.g., ddg_spice_dictionary_definition();. The benefit of doing that is that we know for sure that the Spice failed, and we can do
 	# something about it (we know that the Spice failed because it should return Spice.failed('...') when the parameters are not valid).
-	if($self->callback) {
-		$cfg .= "\tproxy_intercept_errors on;\n";
-		$cfg .= "\terror_page 301 302 303 403 404 500 502 503 504 =200 /js/failed/".$self->callback.";\n";
-	}
+        if($self->callback) {
```

Comment:
Yeah, though I think I would do it outside of the `if` so it could be used on 208-209 as well.  Maybe even put it above on 139 where the other calls are stored.